engine/include/cubos/engine/input/plugin.hpp

```diff
@@ -0,0 +1,26 @@
+/// @file
+/// @brief Input plugin entry point.
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+#include <cubos/engine/input/input.hpp>
+
+namespace cubos::engine
+{
+    /// Plugin which adds input handling to CUBOS.
+    ///
+    /// @details
```

Comment:
Check out the `Scene` plugin docs. This documentation should make it clear that a bridge is being added for loading JSON `InputBindings` with the extension `.bind`